challenges/dog-photo-gallery/script.js

```diff
@@ -0,0 +1,20 @@
+document.getElementById('btn-click').addEventListener('click', () => {
+    fetch("https://dog.ceo/api/breeds/image/random").then(Response => Response.json())
+    .then(data =>{
+        console.log(data)
+        let photoLink = data.message;
```

Comment:
These variables are being declared with `let`, but we generally prefer `const` instead of `let` wherever possible.

Is there a reason these need to be declared with `let`? If so, what's the reason? If not, why do we prefer `const`?